<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditDosisInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Input Dosis Pupuk</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>061238300_1564305222-iStock-684977254.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Prediksi Produksi Pupuk Berdasarkan Dosis Pupuk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTampilModelRegresi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tampilkan Model Regresi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTampilModelRegresi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnprosesInputDosisPupuk">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Proses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTampilkanKorelasi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tampilkan Korelasi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTampilkanKontribusi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tampilkan Kontribusi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKontribusi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTampilkanGrafik">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tampilkan Grafik</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Hasil Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHasilModel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKorelasi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>lineEditDosisInput</zorder>
   <zorder>label</zorder>
   <zorder>label_9</zorder>
   <zorder>btnTampilModelRegresi</zorder>
   <zorder>lineEditTampilModelRegresi</zorder>
   <zorder>btnprosesInputDosisPupuk</zorder>
   <zorder>btnTampilkanKorelasi</zorder>
   <zorder>btnTampilkanKontribusi</zorder>
   <zorder>lineEditKontribusi</zorder>
   <zorder>btnTampilkanGrafik</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEditHasilModel</zorder>
   <zorder>lineEditKorelasi</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
